src/app/components/feed/feeds.factory.js

```diff
@@ -29,6 +29,7 @@
       this.isPublisher = attrs.isPublisher || false;
       this.isLocalScreen = attrs.isLocalScreen || false;
       this.isIgnored = attrs.ignored || false;
+      this.notifierTimeout = 0;
```

Comment:
I wouldn't discard more notifications in the future, so I'd use a less generic name. Let's say something like `speakMutedTimeout`